packages/react/menu/src/Menu.tsx

```diff
@@ -611,8 +611,6 @@ const MenuContentImpl = React.forwardRef<MenuContentImplElement, MenuContentImpl
                       if (isKeyDownInside && !isModifierKey && event.key.length === 1) {
                         handleTypeaheadSearch(event.key);
                       }
-                      // menus should not be navigated using tab key so we prevent it
-                      if (event.key === 'Tab') event.preventDefault();
```

Comment:
Because of events bubbling through portals, this was preventing tab keypress in `Dialog`s opened from within a dropdown.

I'm not sure why we ever needed this though because menus use `RovingFocusGroup` so everything will be `tabindex={-1}` apart from the active item so tab keypress wouldn't do anything anyway.

Reply:
Good catch!

> I'm not sure why we ever needed this line?

> Menus use RovingFocusGroup so everything will be tabindex="-1" apart from the active item meaning tab keypress wouldn't do anything anyway.

This is only true if we’re trapping focus, otherwise tabbing will skip us to the next tabbable element outside the list e.g. when non-modal or in a submenu.

https://user-images.githubusercontent.com/11708259/149922836-2e58a4bf-de24-45d2-a4e3-90b6925248bb.mp4

Reply:
> …but I'm not sure why we ever needed this line?

If my memory isn't failing me, I remember discussing this with you and we found said that if it was a requirement from the component to behave as a single tab-stop (and comparing to native ones) that tab doesn't do anything, even though it was kinda working out of the box with roving, we thought it was too implicit and therefore we decided to be explicit about it.

If we wanted to still stick with this rationale, we could still do that explicitly but with a DOM event rather than react event?

Reply:
> we thought it was too implicit and therefore we decided to be explicit about it.

Your memory is so much better than mine. I don't recall any of that 😅  I'll add the `if (isKeyDownInside && event.key === 'Tab')` as mentioned above because that variable already exists.